buildspecs/pr-build-ubuntu.yml

```diff
@@ -44,9 +44,7 @@ phases:
       - DEBIAN_FRONTEND=noninteractive
         
       - apt-get update -y | tee -a $BUILD_LOG
-      - apt-get install -y make dpkg-dev devscripts debhelper golang | tee -a $BUILD_LOG
-      - which go
-      - go version
+      - apt-get install -y make dpkg-dev wget devscripts debhelper | tee -a $BUILD_LOG
```

Comment:
The ```GOPATH``` in this file is probably not being used now?

We should also update https://github.com/aws/amazon-ecs-agent/blob/master/buildspecs/merge-build-ubuntu.yml with these updates to make builds available in merges to main branch

Reply:
I'll create a follow up PR to fix these up.  TODO Will link here once it's ready.